shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -452,7 +474,7 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
           GpuColumnVector.from(table, selectedAttributes.map(_.dataType).toArray)
         }
       case _ =>
-        throw new IllegalStateException("I don't know how to convert this batch")
+        throw new IllegalStateException("Unknown batch type")
```

Comment:
nit: if we are changing it so it is a more proper error message lets add in at least a little info so someone can debug it, if it does happen.

```
case other =>
    throw new IllegalStateException(s"Unknown batch type ${other.getClass}")
```